### ISSUE TITLE
Having problem opening "Part 2 Forward Propagation.ipynb", "Part 3 Gradient Descent.ipynb" and "Part 4 Backpropagation.ipynb" via Jupyter notebook

### ISSUE THREAD

**Issue by danrol** [stephencwelch/Neural-Networks-Demystified#31]
Hello those files won't open via jupyter notebook
![image](https://user-images.githubusercontent.com/6819208/69129236-a8c48b00-0ab6-11ea-81f2-dd4f650106ea.png)

Windows 10 1809 home edition. Anaconda 4.7.12
![image](https://user-images.githubusercontent.com/6819208/69129415-f3460780-0ab6-11ea-8b50-8e7b4cbdd66a.png)
Thanks in advance

**Comment by gargeesuresh** [comment]
I'm facing the same issue here

**Comment by DidiRaggio** [comment]
This isn't really a fix, but I copied the raw code from each of the non working jupyter notebooks into empty notebooks and that seems to work fine. Perhaps it is an issue with the file permissions, because they are all valid JSON. 
Hope this helped.
This worked fine for me :)

**Comment by const-bon** [comment]
@danrol @gargeesuresh @DidiRaggio @stephencwelch The issue should be resolved now.
https://github.com/stephencwelch/Neural-Networks-Demystified/pull/33
